readme.md

```diff
@@ -137,7 +145,9 @@ Unless the [`shell`](#shell) option is used, no shell interpreter (Bash, `cmd.ex
 Returns a [`child_process` instance](https://nodejs.org/api/child_process.html#child_process_class_childprocess) which:
   - is also a `Promise` resolving or rejecting with a [`childProcessResult`](#childProcessResult).
   - exposes the following additional methods and properties.
+The spawned process can be canceled with the `.cancel()` method on the promise, which causes the promise to reject an error with a `.isCanceled = true` property, provided the process gets canceled. The process will not be canceled if it has already exited.
```

Comment:
This was removed in previous commits. It probably appeared because of the rebase, but you can safely remove it.

Reply:
Done: e2d589df448152eefd4eac35c845dc2d4460af69